libass/ass.c

```diff
@@ -469,7 +469,14 @@ void ass_process_force_style(ASS_Track *track)
                 || ass_strcasecmp(track->styles[sid].Name, style) == 0) {
                 target = track->styles + sid;
                 PARSE_START
-                    STRVAL(FontName)
+                    //STRVAL(FontName) with graceful handling of strdup failure
```

Comment:
`STRVAL` is also used in regular Event and style parsing, where there isn't any previous value to fallback to. Adding it would harm either though (besides possibly some minor(?) performance  penalties)
Or are you suggesting to **re**define `STRVAL` before parsing style overrides ?

Reply:
I think this variant is suitable for regular processing too. It's not worth to have two different variants of error handling.

Reply:
Done.